mono/metadata/object-internals.h

```diff
@@ -46,7 +46,7 @@
 
 /* Use this as MONO_CHECK_ARG_NULL_HANDLE (arg,) in functions returning void */
 #define MONO_CHECK_ARG_NULL_HANDLE(arg, retval) do { 		\
-	if (MONO_HANDLE_IS_NULL (arg))				\
+	if (G_UNLIKELY (MONO_HANDLE_IS_NULL (arg)))		\
```

Comment:
"likely" has bounced here, somewhat by accident.
Arguably it should  be in MONO_HANDLE_IS_NULL.

Reply:
I don't think it makes sense for `likely`/`unlikey` to be part of `MONO_HANDLE_IS_NULL` - different circumstances would have different likelihoods

Reply:
Agreed, though barely. Pretty few paths will have l likely(null), I think.  But I understand, everything is context relative.